core/chains/evm/config/toml/defaults/Linea_Sepolia.toml

```diff
@@ -0,0 +1,14 @@
+ChainID = '59141'
+FinalityDepth = 900
+NoNewHeadsThreshold = '0'
+
+[GasEstimator]
+EIP1559DynamicFees = true
+BumpPercent = 10
```

Comment:
Since this is testnet & also DF we can have a little higher Bumping percentage @HelloKashif

Reply:
Sounds good. 20% is the default so we can remove the field and let if fallback to that?
cc @davidcauchi